docs/user-tutorial/data-diagnosis.md

```diff
@@ -147,15 +156,20 @@ The criterion used for this rule, which indicates how to compare the data with t
 
 #### `store`
 
-True if the current rule is not used alone to filter the defective machine, but will be used by other subsequent rules. False(default) if this rule is used to label the defective machine directly.
+- True: the current rule is not used alone to filter the defective machine, but will store some temporary data which will be used by other subsequent rules.
```

Comment:
True: this label is used to store metrics which will be used by other subsequent rules.